pywps/inout/basic.py

```diff
@@ -495,15 +498,16 @@ def _openurl(href, data=None):
 
         return req
 
-    @staticmethod
-    def max_input_size():
+    def max_size(self):
         """Calculates maximal size for input file based on configuration
         and units.
 
-        :return: maximum file size in bytes
+        :return: maximum file size in bytes and megabytes
         """
         ms = config.get_config_value('server', 'maxsingleinputsize')
-        return config.get_size_mb(ms) * 1024**2
+        mb_size = config.get_size_mb(ms)
+        byte_size = mb_size * 1024**2
+        return byte_size, mb_size
```

Comment:
I think it's unexpected that a function would return the same output with two different units. I suggest PyWPS enforces some kind of developer convention, that whenever file size is described by float or int, it's in units of bytes.

The string formatting can then be done with something like that: https://stackoverflow.com/questions/1094841/get-human-readable-version-of-file-size

Reply:
If you're interested, I have a function or two I built based on some of those SO responses. https://github.com/Ouranosinc/miranda/blob/1808dd7de48a650527b0968326ff051d137077a1/miranda/storage.py#L234 and https://github.com/Ouranosinc/miranda/blob/1808dd7de48a650527b0968326ff051d137077a1/miranda/storage.py#L273. Feel free to copy and adapt.

Reply:
Right  👍  ... I look into this.

Reply:
Ok to use [humanize](https://pypi.org/project/humanize/)? We might want to do this more often ...

Reply:
I had no idea this existed. I can definitely find a use for this library elsewhere. Great find!

Reply:
👍